app/helpers.php

```diff
@@ -221,3 +221,19 @@ function _fmt($line, array $replace)
         return $line;
     }
 }
+
+if (!function_exists('__trans_choice')) {
+    /**
+     * Translates the given message based on a count from json key.
+     *
+     * @param $key
+     * @param $number
+     * @param array $replace
+     * @param null $locale
+     * @return string
+     */
+    function __trans_choice($key, $number, array $replace = [], $locale = null)
```

Comment:
Can we change this to `__tc`, just to make it shorter in the templates?

Reply:
if i move all in php files then there is a dedicated function called trans_choice()

Reply:
Ok. Don't change anything just yet in your PR, let me do some research